integration-cli/docker_cli_run_test.go

```diff
@@ -531,6 +531,14 @@ func (s *DockerSuite) TestRunNoDupVolumes(c *check.C) {
 			c.Fatalf("Expected 'duplicate mount point' error, got %v", out)
 		}
 	}
+	// create failed should have create volume mountstr1 or mountstr2
+	// we should remove mountstr1 or mountstr2 successfully
+	out, _ := dockerCmd(c, "volume", "ls")
+	if strings.Contains(out, mountstr1) {
```

Comment:
@cpuguy83  this is to test if we can remove the volume successfully base on the issue #22093